### ISSUE TITLE
Allow Arrow Keys in Notes Field

### ISSUE THREAD

**Issue by jonheal** [VCVRack/Rack#727]
OS (if applicable): Windows 10
Version (or "dev" if compiling from source): 5.1.1
If not too late for version .6, add functionality to enable arrow keys on keyboard to move caret around in text of Notes module.

**Comment by AndrewBelt** [comment]
Have you tried this too? Left/right works. Up/down is hard so I haven't added it.

**Comment by jonheal** [comment]
Yeah, up and down was what I was looking for. Oh, well. Thanks anyway.

--
Jon Heal
http://www.theheals.org/
Have a nice day!
